lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -34,7 +34,8 @@
 #include "circt/Dialect/FIRRTL/FIRRTLVisitors.h"
 #include "circt/Dialect/FIRRTL/Passes.h"
 #include "mlir/IR/ImplicitLocOpBuilder.h"
-#include "llvm/Support/Parallel.h"
+#include "mlir/IR/Threading.h"
+#include "llvm/ADT/APSInt.h"
```

Comment:
This `APSint.h` include is now necessary since `Parallel.h` was pulling this in and there's an `arg.value()` call that returns an `APSInt` and compilation wants a complete return type.

Reply:
+1 thanks